README.md

```diff
@@ -10,208 +10,124 @@
-            <td>Intel Processor Graphics, including Intel HD Graphics and Intel Iris Graphics</td>
-        </tr>
-        <tr>
-            <td>NPU</td>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/npu-device.html">Intel NPU</a></td>
-            <td><b><i><a href="./src/plugins/intel_npu">openvino_intel_npu_plugin</a></i></b></td>
-            <td>Intel® Core™ Ultra Processors</td>
-        </tr>
-    </tbody>
-</table>
-
-OpenVINO™ Toolkit also contains several plugins which simplify loading models on several hardware devices:
-<table>
-    <thead>
-        <tr>
-            <th>Plugin</th>
-            <th>Library</th>
-            <th>Short Description</th>
-        </tr>
-    </thead>
-    <tbody>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/auto-device-selection.html">Auto</a></td>
-            <td><b><i><a href="./src/plugins/auto">openvino_auto_plugin</a></i></b></td>
-            <td>Auto plugin enables selecting Intel device for inference automatically</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/automatic-batching.html">Auto Batch</a></td>
-            <td><b><i><a href="./src/plugins/auto_batch">openvino_auto_batch_plugin</a></i></b></td>
-            <td>Auto batch plugin performs on-the-fly automatic batching (i.e. grouping inference requests together) to improve device utilization, with no programming effort from the user</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/hetero-execution.html">Hetero</a></td>
-            <td><b><i><a href="./src/plugins/hetero">openvino_hetero_plugin</a></i></b></td>
-            <td>Heterogeneous execution enables automatic inference splitting between several devices</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/multi-device.html">Multi</a></td>
-            <td><b><i><a href="./src/plugins/auto">openvino_auto_plugin</a></i></b></td>
-            <td>Multi plugin enables simultaneous inference of the same model on several devices in parallel</td>
-        </tr>
-    </tbody>
-</table>
+Welcome to OpenVINO™, an open-source software toolkit for optimizing and deploying deep learning models.
 
-## License
-OpenVINO™ Toolkit is licensed under [Apache License Version 2.0](LICENSE).
-By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
+- **Inference Optimization**: Boost deep learning performance in computer vision, automatic speech recognition, natural language processing, and many other common tasks.
```

Comment:
Should we also have a separate bullet to mention different APIs: Python, C/C++, and JS?
@mlukasze, @ilya-lavrenov